frontend/packages/operator-lifecycle-manager/src/components/operand/index.tsx

```diff
@@ -130,7 +130,6 @@ const tableColumnClasses = [
   '',
   '',
   classNames('pf-m-hidden', 'pf-m-visible-on-sm', 'pf-u-w-16-on-lg'),
-  classNames('pf-m-hidden', 'pf-m-visible-on-lg'),
   classNames('pf-m-hidden', 'pf-m-visible-on-xl'),
   classNames('pf-m-hidden', 'pf-m-visible-on-2xl'),
```

Comment:
I wonder if we should adjust some of these other classes since we have a little more space.

Reply:
I pulled this down to test, and I think it's good as-is. Disregard.